docs/en/latest/plugins/sls-logger.md

```diff
@@ -46,7 +46,7 @@ It might take some time to receive the log data. It will be automatically sent a
 | access_key_id     | True     | AccessKey ID in Alibaba Cloud. See [Authorization](https://www.alibabacloud.com/help/en/log-service/latest/create-a-ram-user-and-authorize-the-ram-user-to-access-log-service) for more details.                                                                     |
 | access_key_secret | True     | AccessKey Secret in Alibaba Cloud. See [Authorization](https://www.alibabacloud.com/help/en/log-service/latest/create-a-ram-user-and-authorize-the-ram-user-to-access-log-service) for more details.                                                                 |
 | include_req_body  | True     | When set to `true`, includes the request body in the log.                                                                                                                                                                                       |
-| name              | False    | Unique identifier for the batch processor.                                                                                                                                                                                                      |
+| name              | False    | Unique identifier for the batch processor. If you use Prometheus to monitor APISIX metrics, the name is exported in `apisix_batch_process_entries`.                                                                                                                                                                                                      |
```

Comment:
it sometimes says "for the batch processor", some other times it's "for the logger" or "to identify the logger"; they should be consistent since they're all for the same purpose.